runtime/src/main/java/org/corfudb/runtime/LogReplicationUtils.java

```diff
@@ -58,6 +59,9 @@ public static void subscribe(@Nonnull LogReplicationListener listener, @Nonnull
                                  int bufferSize, CorfuStore corfuStore) {
         log.info("Client subscription process has started.");
 
+        // Ensure status table is opened here in case FullSyncExecutor is late to do so.
```

Comment:
Did you hit into this policy demo testing or some other use-case?

Reply:
This is actually the listener for the logical group use case, so yes unrelated to the demo work.